core/src/data/clientGeoJsonSource.cpp

```diff
@@ -44,15 +44,15 @@ ClientGeoJsonSource::ClientGeoJsonSource(const std::string& _name, const std::st
         std::regex r("^(http|https):/");
         std::smatch match;
         if (std::regex_search(_url, match, r)) {
-            startUrlRequest(_url,
+            m_platform->startUrlRequest(_url,
```

Comment:
It is already included here https://github.com/tangrams/tangram-es/pull/1263/files/7ebd47ea2232dab4febaf26c21363126738046b3#diff-3cdb39a640f4b049c126ddf4f998bffaR6.